### ISSUE TITLE
Events: Customise Elfsight widget

### ISSUE THREAD

**Issue by wei2912** [AdvisorySG/dawn-advisory-theme#140]
Elfsight widget should be customised and refined further with the help of Programmes.

**Comment by Hackin7** [comment]
If the Elfsight widget is too difficult to customise to our liking, we may want to either create or find another solution to show Advisory's Upcoming Events

**Comment by Hackin7** [comment]
I have also experimented with doing our own custom styling on top of the elfsight widget

In this case I felt that the cards look a bit too flat, so we could add a box shadow. Box Shadow code could be retrieved from [here](https://getcssscan.com/css-box-shadow-examples)

![Image](https://user-images.githubusercontent.com/18213722/234017036-5600bc23-4f34-4033-8820-39925cf4eed9.png)

Elfsight has a place where we can put custom CSS, to tweak the UI to our liking
For now I've
1. added a box shadow to the cards -> readability purposes
2. added padding for the error text
3. Added padding & margins to line everything up nicely

```
div{align: center}

.eapp-events-calendar-layout-pastEvents{
  margin-top: 0;
}
.eapp-events-calendar-layout-pastEventsTitle {
    margin-left: 10px;
}
.eapp-events-calendar-grid-component{
  padding: 10px;
}
.eapp-events-calendar-grid-item-component{
    box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
}
.eapp-events-calendar-error-text {
    margin-top: 1em;
}
```

![Image](https://user-images.githubusercontent.com/18213722/234022337-7500a4db-37af-4b3d-beba-8cfb47121210.png)
